Update Vnet FAQ re: DHCP Unicast Support

ryanbor opened the pull request:
DHCP via Unicast (such as a DHCP relay) is unsupported in Azure. Updated docs to reflect this support.

azurecla commented:
Hi __@xpouyat__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (xpouyat). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ryanbor committed:
Update Vnet FAQ re: DHCP Unicast Support

DHCP via Unicast (such as a DHCP relay) is unsupported in Azure. Updated docs to reflect this support.
